scripts/jenkins/ardana/gerrit/gerrit_review.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import argparse
+
+from pygerrit2 import GerritRestAPI, GerritReview, HTTPDigestAuthFromNetrc
+
+GERRIT_URL = 'https://gerrit.suse.provo.cloud/'
+
+
+def gerrit_review(change_no, patch=None, label=None, vote=1, message=''):
+    auth = HTTPDigestAuthFromNetrc(url=GERRIT_URL)
+    rest = GerritRestAPI(url=GERRIT_URL, auth=auth, verify=False)
+    change = rest.get("/changes/?q={}".format(change_no))[0]
+    print "Posting {}: {} review for change: '{}'".format(label, vote,
```

Comment:
(nit) we should probably target python 3 for new scripts given it's impending EOL.